Library/Homebrew/PATH.rb

```diff
@@ -39,12 +39,12 @@ def insert(index, *paths)
     self
   end
 
-  sig { params(block: T.proc.params(arg0: String).returns(T::Boolean)).returns(T.self_type) }
+  sig { params(block: T.proc.params(arg0: String).returns(BasicObject)).returns(T.self_type) }
```

Comment:
This was an incompatible override, see https://github.com/sorbet/sorbet/blob/662a1a4e956ac74a1b2c22f33f590f38b92ecb74/rbi/core/enumerable.rbi#L2016 (also true for `reject` below)

We cannot explicitly mark this as `override` though, bc the overall return type is incompatible.